library/general/requiredlabels/template.yaml

```diff
@@ -38,31 +38,42 @@ spec:
                       the regular expression.
   targets:
     - target: admission.k8s.gatekeeper.sh
-      rego: |
-        package k8srequiredlabels
+      code:
+      - engine: K8sNativeValidation
+        source:
+          validations:
+          - expression: '(has(object.metadata) && variables.params.labels.exists_one(entry, has(object.metadata.labels) && entry.key in object.metadata.labels))'
```

Comment:
`exists_one()` is wrong. This check requires exactly ONE required label to be present (not zero, not more than one). `all()` is the correct check, since all required labels must be present.

Reply:
Oh Yes! missed on that. Good catch.

Reply:
I think I finally got the CEL code right. Let me know how it looks now.